js/binance.js

```diff
@@ -3021,7 +3021,10 @@ module.exports = class binance extends Exchange {
                     if (quoteOrderQty !== undefined) {
                         request['quoteOrderQty'] = this.decimalToPrecision (quoteOrderQty, TRUNCATE, precision, this.precisionMode);
                     } else if (price !== undefined) {
-                        request['quoteOrderQty'] = this.decimalToPrecision (amount * price, TRUNCATE, precision, this.precisionMode);
+                        const amountString = this.numberToString (amount);
+                        const priceString = this.numberToString (price);
+                        const quoteOrderQuantity = this.parseNumber (Precise.stringMul (amountString, priceString));
```

Comment:
again, `decimalToPrecision` should receive a string so we don't need to convert it here